hardhat/dist/constants.d.ts

```diff
@@ -5,7 +5,6 @@ export declare const GANACHE_PORT: string | undefined;
 export declare const GANACHE_URL: string;
 export declare const ETHEREUM_RPC_URL: string;
 export declare const FORKED_BLOCK: number | undefined;
-export declare const ZERO_ADDRESS = "0x0000000000000000000000000000000000000000";
```

Comment:
this and the change in `constants.js` should have been included in the commit that removed the ZERO_ADDRESS constant from the TS code